smoketest/scripts/cli/test_interfaces_bridge.py

```diff
@@ -35,6 +35,7 @@ def setUp(self):
 
         self._base_path = ['interfaces', 'bridge']
         self._interfaces = ['br0']
+        self._mirror_interfaces = ['bond1']
```

Comment:
Who creates bond1?

Reply:
The code for creating 'bond1' is below. I share 'bond1'`